### PR TITLE
fix time-sweep #144

### PR DESCRIPTION
**Pull request by nicolaskolbenschlag**
Signed-off-by: nicolaskolbenschlag <nicolas.kolbenschlag@googlemail.com>

### CODE REVIEW

File: Apps/frontend/src/const.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ export const LINE_THICKNESS_BIG = 0.008;
 export const NUM_CHANNELS = 10;
 export const MIN_SWEEP = 0.1; // <= 1
 export const MAX_SWEEP = 2.0; // >= 1
+export const DEFAULT_STEP_SIZE = 1.0;
```
Comment (jandegen) [inline Apps/frontend/src/const.js:37]: Please add js doc like /** Used for timesweep to ... */

File: Apps/frontend/src/components/Waves.svelte [hunk 2]
```diff
@@ -91,9 +92,9 @@
       }
       xLast[channelIndex] = xNew;
 
-      let sweep = $timeSweep[channelIndex] / 5.0 - 1.0;
-      let fac = sweep < 0 ? MIN_SWEEP : MAX_SWEEP;
-      let delta = fac * sweep + 1.0;
+      // time sweep
+      let sweep = $timeSweep[channelIndex] / 5.0 - 1.0;// in [-1,1]
```
Comment (jandegen) [inline Apps/frontend/src/components/Waves.svelte:96]: Please improve documentation about calculations. Why do we need to do / 5.0 -1

**Review by jandegen — CHANGES_REQUESTED** [review]
Seems fine and working but please add documention

File: Apps/frontend/src/components/Waves.svelte (hunk 2 above)
Comment (nicolaskolbenschlag) [reply Apps/frontend/src/components/Waves.svelte:96]: Documentation for this can be found in the Wiki

**Review by jandegen — APPROVED** [review]

**Pull request merged**